mycli/output_formatter/delimited_output_adapter.py

```diff
@@ -0,0 +1,23 @@
+import contextlib
+import csv
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+from .preprocessors import override_missing_value, bytes_to_string
+
+supported_formats = ('csv',)
```

Comment:
Hmm, if we add `tsv` to the list, it's not really different from `csv` because there is nothing in the adapter that treats `tsv` differently. In other words, it's up to output_formatter to know to associate `tsv` with `\t`.

I wonder if the `delimiter_adapter` should take a format argument that maps format to delimiter.